openpype/hooks/pre_copy_last_published_workfile.py

```diff
@@ -31,16 +32,12 @@ def execute(self):
         2- Check if workfile in work area doesn't exist
         3- Check if published workfile exists and is copied locally in publish
         4- Substitute copied published workfile as first workfile
+           with incremented version by +1
 
         Returns:
             None: This is a void method.
         """
 
-        sync_server = self.modules_manager.get("sync_server")
```

Comment:
So this will happen all the time even if sync server is not available or enabled?

Reply:
I think this is a mistake.